content/en/integrations/vmware_tanzu_application_service.md

```diff
@@ -0,0 +1,163 @@
+
+```shell
+# Redirect Stdout/Stderr to port 10514
+cf set-env app01 STD_LOG_COLLECTION_PORT 10514
+# Configure the Agent to listen to port 10514
+cf set-env app01 LOGS_CONFIG '[{"type":"tcp","port":"10514","source":"java","service":"app01"}]'
+```
+
+##### Notification in case of misconfigured proxy
+
+For Agent version 6.12 or greater, when using a [proxy configuration](/agent/logs/proxy/) with the buildpack, a verification is made to check if the connection can be established. Log collection is started depending on the result of this test.
+
+If the connection fails to be established and the log collection is not started, an event like the one below is sent to your Datadog event explorer. Set up a monitor to track these events and be notified when a misconfigured Buildpack is deployed:
+
+{{< img src="integrations/cloud_foundry/logs_misconfigured_proxy.png" alt="An event in Datadog with the title Log endpoint cannot be reached - Log collection not started and a message stating that a TCP connection could not be established"  >}}
+
+### Tags
+
+In order to add custom tags to your application, set the `DD_TAGS` environment variable through the `manifest.yml` file or the CF CLI command:
+
+```shell
+# set the environment variable
+cf set-env <YOUR_APP> DD_TAGS key1=value1,key2=value2
+# restage the application to make it pick up the new environment variable and use the new tags
+cf restage <YOUR_APP>
+```
+
+{{< /site-region >}}
+
+### DogStatsD
+
+You can use [DogStatsD][33] to send custom application metrics to Datadog. See [Metric Submission: DogStatsD][5] for more information. There is a list of [DogStatsD libraries][14] compatible with a wide range of applications.
+
+## Monitor your VMware Tanzu Application Service cluster
+
+Use the [VMware Tanzu installation and configuration][9] guide to install the integration through the Tanzu Ops Manager. For manual setup steps, read the [Monitor your VMware Tanzu Application Service cluster][13] section in the manual setup guide.
+
+## Data Collected
+
+### Metrics
+
+The following metrics are sent by the Datadog Firehose Nozzle and are prefixed with `cloudfoundry.nozzle`. The Datadog Agent release sends metrics from any Agent checks you configure in the Director runtime configuration, and [system][24], [network][25], [disk][26], and [NTP][27] metrics by default.
+
+The Datadog Firehose Nozzle only collects CounterEvents (as metrics, not events), ValueMetrics, and ContainerMetrics; it ignores LogMessages and Errors.
+
+Your specific list of metrics may vary based on the PCF version and the deployment. Datadog collects counter and gauge metrics emitted from the [Loggregator v2 API][28]. See [Cloud Foundry Component Metrics][29] for a list of metrics emitted by default.
+
+{{< get-metrics-from-git "cloud_foundry">}}
+
```

Comment:
Should we add a section for ### Events here since we discuss events appearing the Event Explorer and maybe add a Further Reading section?

Reply:
I think you're referring to the mention of the Events Explorer here:
https://github.com/DataDog/documentation/pull/14955/files#r1040215458

In this case, the event is only generated when the connection is failed, and if the integration is working these events wouldn't be expected.